ReferenceAdapter/src/main/java/com/chartboost/mediation/referenceadapter/sdk/ReferenceFullscreenActivity.kt

```diff
@@ -149,6 +175,50 @@ class ReferenceFullscreenActivity : AppCompatActivity() {
                 }
             })
 
+            if (adType == ReferenceFullscreenAd.ReferenceFullscreenAdFormat.REWARDED_INTERSTITIAL) {
+                val timerView = binding.referenceFullscreenTimerview
+                val closeButton = binding.referenceFullscreenClosebutton
+                val params = FrameLayout.LayoutParams(
+                    FrameLayout.LayoutParams.WRAP_CONTENT,
+                    FrameLayout.LayoutParams.WRAP_CONTENT
+                ).apply {
+                    gravity = Gravity.TOP or Gravity.END
+                    setMargins(0, 0, 10, 0)
+                }
+
+                var timer: CountDownTimer? = null
```

Comment:
hmmmm you _could_ make this non null by making the timer and not starting it but then.... it'll immediately get overwritten soooooo maybe not lol